lib/modules/wdpa/dopa_importer.rb

```diff
@@ -0,0 +1,32 @@
+module Wdpa::DopaImporter
+  DOPA_LIST = "#{Rails.root}/lib/data/seeds/dopa4_pas_list.csv"
+  extend self
+
+  def import
+    ActiveRecord::Base.transaction do
+      # Clear all previously set DOPA areas
+      ProtectedArea.update_all(is_dopa: false)
+      logger = Logger.new(STDOUT)
+
+      pas_to_update = []
+
+      # Read the CSV to get the WDPA IDs
+      CSV.foreach(DOPA_LIST, headers: true) do |row| 
+        # Find the associated Protected Area with each WDPA ID mentioned and update the is_dopa column to true
+        dopa_pa = ProtectedArea.where('reported_area > 0.5e1')
+                  .or(ProtectedArea.where('reported_marine_area > 0.5e1'))
```

Comment:
Why is this check necessary? I think we would need to find the PAs by the WDPA ID only.

Reply:
https://unep-wcmc.codebasehq.com/projects/pp-refresh/tickets/115#update-63761477 I was under the impression that we had to impose this restriction because of the DOPA Explorer's own limitation.

Reply:
I think that already implies that the CSV provided will only contain the WDPA IDs of the areas they show; so those kind of areas are already been excluded at the point of generating the CSV